scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationServiceTest.java

```diff
@@ -41,15 +41,24 @@ void testCodeGen() {
                  + "interface PeelBanana extends ActivityTemplate {}\n"
                  + "export const ActivityTemplates = {\n"
                  + "  PeelBanana: function PeelBanana(\n"
-                 + "    name: string\n"
                  + "    args: {\n"
                  + "      duration: Duration\n"
                  + "      fancy: { subfield1: string, subfield2: { subsubfield1: Double, }[], }\n"
                  + "      peelDirection: (\"fromTip\" | \"fromStem\")\n"
                  + "    }): PeelBanana {\n"
-                 + "      return { name, activityType: 'PeelBanana', args };\n"
+                 + "      return { activityType: 'PeelBanana', args };\n"
                  + "    },\n"
                  + "}\n"
+                 + "declare global {\n"
+                 + "  var ActivityTemplates: {\n"
+                 + "    PeelBanana: (\n"
+                 + "      args: {\n"
+                 + "        duration: Duration\n"
```

Comment:
Duration isn't a JavaScript entity, so this will surely fail at runtime/typecheck time.

Reply:
I've added a `Duration` class to the DSL, so instead of `24 * 60 * 60 * 1000 * 1000`, the syntax will now be `Duration.ofHours(24)`.

Reply:
As @dyst5422 pointed out, there's a new feature coming down the pipeline in typescript that adds a `Duration` class to the standard library. It is worth considering using that for the scheduling dsl. However, I've decided that I want to give it some careful thought, and make it a separate PR. I've dropped the commit that changes the type of Duration, so it is now just a type alias for `number`.